atuin/src/command/client/history.rs

```diff
@@ -51,6 +51,10 @@ pub enum Cmd {
         #[arg(long)]
         cmd_only: bool,
 
+        /// explicitely include deleted
+        #[arg(long)]
+        include_deleted: bool,
```

Comment:
I don't _think_ there is ever a time when the user would want to include deleted records in outputs, so I think we're safe to only "include deleted" from sync functions - elsewhere, just default to false.